app/views/layouts/application.html.erb

```diff
@@ -12,16 +12,22 @@
   <body>
   <nav class="navbar navbar-expand-lg navbar-light fixed-top" id="mainNav">
             <div class="container">
-                <a class="navbar-brand js-scroll-trigger" href="#page-top">I Scream Arcade</a>
+                <a class="navbar-brand js-scroll-trigger" href="/">I Scream Arcade</a>
```

Comment:
This part can be changed to a logo later but for now when you click on it, it will go to the homepage